### ISSUE TITLE
Typo in the Eq 5?

### ISSUE THREAD

**Issue by a-jahani** [cogaplex-bts/bts#2]
Dear Authur,

Thank you for sharing your codes.

I have a question about equation 5:

![Screenshot from 2019-08-28 19-53-29](https://user-images.githubusercontent.com/13435607/63904292-b1cd2100-c9cd-11e9-9fa7-594631343765.png)

Do you meant h= beta x logy - beta x log y* ?
The way you described in the paper, betas are going to cancel out each other and have no effect!

Please fix me if I am wrong.
Thanks,
Ali

**Comment by cogaplex-bts** [comment]
You are right.
By math, they will simply be cancelled due to the logarithms.
That ugly trick is just our empirical choice and will be improved in next release.
In fact, we are preparing new release due to inconsistency (small difference in performance) between currently relased model and the paper.
Thanks for kindly pointing this. =)

**Comment by a-jahani** [comment]
Thanks for the code. I like the idea:) :)

Now that I'm looking at the code there is no beta implemented in the code. I was wondering if you removed beta in this provided code. I'm trying to reproduce your result that you submitted to the benchmark. Please let me know if something is missing for that.
I was also wondering if the model provided for eigen is the same model as the one you submitted to KITTI benchmark? If not can you mention the changes you did for the model you submitted to the [benchmark](http://www.cvlibs.net/datasets/kitti/eval_depth_detail.php?benchmark=depth_prediction&result=eaa9582a2aab90e373bb9d5ee5b157cd1b8f7dc3) ?


If you don't mind here is some inconsistencies I noticed if you wanna edit your paper:
In default parameters for eigen split(i.e. batch size and rotate degree)

and also I think n3 should be passed to a separate sigmoid (not tanh) and then n1,n2 (after tanh) and n3 (after sigmoid) be passed to L2 normalization. Nice trick to avoid minus of the normals btw :) You just need to update the figure I think.

![image](https://user-images.githubusercontent.com/13435607/64665795-139b7b00-d411-11e9-812d-6056f3e0d99a.png)



Thanks,
Ali

**Comment by cogaplex-bts** [comment]
Hello, nice to hearing from you again. =)
As I mentioned before, I updated the code not to include \beta in the training loss, and updated the model file as well trained with the modified loss function.
As a matter of fact, the model submitted to the KITTI online benchmark is different from trained with Eigen split. The benchmark version was trained with sampled examples (about 57000 rgb and gt pairs) from whole dataset, but parameters are almost same: batch_size 16, num_gpus 4, num_epochs 50, input_height 352, input_width 704, do_kb_crop, do_random_rotate, degree 1.0.
The model for the Eigen split is trained with the parameters in "arguments_train_eigen.txt", only batch_size and num_gpus are different (actually they were 16 and 4, respectively) because I assume most of users will use our code with only one gpu.
Finally, I have to say that this work is still in progress.
I mean, before publication (not a preprint in arXiv) our code and paper will be updated.
This is why we made our code and paper publicly available, this kind of very constructive review from you! =)
Thanks,
Jin Han

**Comment by cogaplex-bts** [comment]
Closing this issue due to inactivity

**Comment by luohongcheng** [comment]
Hi, is the model submitted to the KITTI online benchmark trained on the official **" annotated depth maps data set (14 GB)"**, which only contains 43000 rgb images.

**Comment by cogaplex-bts** [comment]
> Hi, is the model submitted to the KITTI online benchmark trained on the official **" annotated depth maps data set (14 GB)"**, which only contains 43000 rgb images.

**Comment by cogaplex-bts** [comment]
@luohongcheng Yes. The submitted model was finetued using only the official ground truth depth maps with the backbone network pretrained using ImageNet.

**Comment by luohongcheng** [comment]
Would you please share your file lists selected from the official training set? Thanks.
